Tensile/Components/MAC_F32.py

```diff
@@ -80,19 +87,56 @@ def __call__(self, writer, m, innerUnroll):
                     vars["aStr"] = "v[vgprValuA_X{m}_I{iui} + {a}]".format_map(vars)
                     vars["bStr"] = "v[vgprValuB_X{m}_I{iui} + {b}]".format_map(vars)
 
-                    if instruction == "v_fma_f32":
-                        kStr += "v_fma_f32 {cStr}, {aStr}, {bStr}, {cStr}{endLine}".format_map(vars)
+                    instVars = {}
```

Comment:
You can move instVars related code after if dualMacEnable == 1.
It is unnecessary for dualMacEnable == 0.